### PR TITLE
Add PerspectiveManager remote API for Python, Tornado server example

### PR DESCRIPTION
**Pull request by sc1f**
This PR adds the PerspectiveManager class, which parses JSON messages from a client perspective and allows for operations in perspective-python on a dataset in python memory, streamed through a WebSocket to a client perspective operating in the browser. 

This configuration is not subject to browser memory constraints or the 2GB WebAssembly memory limit, and takes advantage of TBB for parallel processing on the server side. Configuration has been tested locally with 10 million rows, streamed with zero lag to the browser.

Additionally, this has the side benefit of allowing us to serve dataframes and Numpy array-based datasets to the client through the python API.

### CODE REVIEW

**Comment by texodus** [comment]
Thanks for the PR!  Looks great!

File: python/perspective/examples/perspective_tornado_server.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+from perspective import Table, PerspectiveManager
+from datetime import date, datetime
+import json
+import tornado.websocket
+import tornado.web
+import tornado.ioloop
+import pandas as pd
+import sys
+import os
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
```
Comment (timkpaine) [inline python/perspective/examples/perspective_tornado_server.py]: i  wouldnt do this

File: python/perspective/perspective/table/manager.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                if callback_id:
+                    self._callback_cache[callback_id] = callback
+            elif callback_id is not None:
+                # remove the callback with `callback_id`
+                del self._callback_cache[callback_id]
+            if callback is not None:
+                # call the underlying method on the Table or View
+                getattr(table_or_view, method)(callback, *msg.get("args", []))
+            else:
+                print("callback not found for remote call {}".format(msg))
```
Comment (timkpaine) [inline python/perspective/perspective/table/manager.py]: avoid prints, use logging if you want to print

File: python/perspective/perspective/table/manager.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                if msg["method"] == "schema":
+                    args = [True]  # make sure schema returns string types
+                if msg["method"] != "delete":
+                    result = getattr(table_or_view, msg["method"])(*args)
+                    post_callback(self._make_message(msg["id"], result))
+                else:
+                    if msg["cmd"] == "view_method":
+                        del self._views[msg["name"]]
+        except Exception as error:
+            print(self._make_error_message(msg["id"], error))
```
Comment (timkpaine) [inline python/perspective/perspective/table/manager.py]: remove print

File: python/perspective/perspective/table/manager.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+            elif callback_id is not None:
+                # remove the callback with `callback_id`
+                del self._callback_cache[callback_id]
+            if callback is not None:
+                # call the underlying method on the Table or View
+                getattr(table_or_view, method)(callback, *msg.get("args", []))
+            else:
+                print("callback not found for remote call {}".format(msg))
+        except Exception as error:
+            print(self._make_error_message(msg["id"], error))
```
Comment (timkpaine) [inline python/perspective/perspective/table/manager.py]: remove print

**Pull request merged**